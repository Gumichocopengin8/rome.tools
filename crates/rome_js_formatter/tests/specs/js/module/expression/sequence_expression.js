a,b

abcdefghijklkmnopqrstufwkdsf,
    abcdefghijklkmnopqrstufwkdsfddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd;

abcdefghijklkmnopqrstufwkdsf,
    abcdefghijklkmnopqrstufwkdsf,
    abcdefghijklkmnopqrstufwkdsf,
    abcdefghijklkmnopqrstufwkdsf,
    abcdefghijklkmnopqrstufwkdsf,
    abcdefghijklkmnopqrstufwkdsf,
    abcdefghijklkmnopqrstufwkdsf,
    abcdefghijklkmnopqrstufwkdsf;