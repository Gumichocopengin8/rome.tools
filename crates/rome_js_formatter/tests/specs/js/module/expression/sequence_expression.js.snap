---
source: crates/rome_js_formatter/tests/spec_test.rs
assertion_line: 223
expression: sequence_expression.js

---
# Input
a,b


abcdefghijklkmnopqrstufwkdsf, abcdefghijklkmnopqrstufwkdsfddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd;

abcdefghijklkmnopqrstufwkdsf, abcdefghijklkmnopqrstufwkdsf, abcdefghijklkmnopqrstufwkdsf, abcdefghijklkmnopqrstufwkdsf, abcdefghijklkmnopqrstufwkdsf, abcdefghijklkmnopqrstufwkdsf, abcdefghijklkmnopqrstufwkdsf, abcdefghijklkmnopqrstufwkdsf;
=============================
# Outputs
## Output 1
-----
Indent style: Tab
Line width: 80
Quote style: Double Quotes
-----
a, b;

abcdefghijklkmnopqrstufwkdsf,
	abcdefghijklkmnopqrstufwkdsfddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd;

abcdefghijklkmnopqrstufwkdsf,
	abcdefghijklkmnopqrstufwkdsf,
	abcdefghijklkmnopqrstufwkdsf,
	abcdefghijklkmnopqrstufwkdsf,
	abcdefghijklkmnopqrstufwkdsf,
	abcdefghijklkmnopqrstufwkdsf,
	abcdefghijklkmnopqrstufwkdsf,
	abcdefghijklkmnopqrstufwkdsf;

